src/sinopiaServerSpoof.js

```diff
@@ -5,27 +5,23 @@ const monographInstanceRt = require('../static/spoofedFilesFromServer/fromSinopi
 const monographWorkRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/MonographWork.json')
 const noteRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/Note.json')
 const parallelTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/ParallelTitle.json')
-const serialRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/SerialFrequency.json')
 const titleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/Title.json')
 const titleNoteRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/TitleNote.json')
 const transcribedTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/TranscribedTitle.json')
 const varTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/VarTitle.json')
 const workTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/WorkTitle.json')
 const workVariantTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/WorkVariantTitle.json')
-const qaLocVocabFrequencies = require('../static/spoofedFilesFromServer/fromQA/frequencies.json')
 const resourceTemplateId2Json = [
   {'id': 'resourceTemplate:bf2:Monograph:Instance', 'json': monographInstanceRt},
   {'id': 'resourceTemplate:bf2:Monograph:Work', 'json': monographWorkRt},
   {'id': 'resourceTemplate:bf2:Note', 'json': noteRt},
   {'id': 'resourceTemplate:bf2:ParallelTitle', 'json': parallelTitleRt},
-  {'id': 'resourceTemplate:bf2:Serial:Frequency', 'json': serialRt},
   {'id': 'resourceTemplate:bf2:Title', 'json': titleRt},
   {'id': 'resourceTemplate:bf2:Title:Note', 'json': titleNoteRt},
   {'id': 'resourceTemplate:bflc:TranscribedTitle', 'json': transcribedTitleRt},
   {'id': 'resourceTemplate:bf2:Title:VarTitle', 'json': varTitleRt},
   {'id': 'resourceTemplate:bf2:WorkTitle', 'json': workTitleRt},
-  {'id': 'resourceTemplate:bf2:WorkVariantTitle', 'json': workVariantTitleRt},
-  {'id': 'vocabulary:bf2:frequencies', 'json': qaLocVocabFrequencies}
+  {'id': 'resourceTemplate:bf2:WorkVariantTitle', 'json': workVariantTitleRt}
```

Comment:
thank you for re-purifying my sinopiaServerSpoof.  :-)

Reply:
👍